.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/micrometer
+    environment:
+      GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError"'
+    docker:
+      - image: circleci/openjdk:8-jdk
+    steps:
+
+      - checkout
+
+      - restore_cache:
+          key: gradle-dependencies-{{ checksum "build.gradle" }}
+
+      - run:
+          name: downloadDependencies
+          command: ./gradlew downloadDependencies --console=plain
+
+      - save_cache:
+          key: gradle-dependencies-{{ checksum "build.gradle" }}
+          paths:
+            - ~/.gradle
+
+      - run:
+          name: run tests
+          command: ./gradlew test
+
+      - run:
+          name: collect test reports
+          command: |
+            mkdir -p ~/micrometer/junit/
+            find . -type f -regex ".*/build/test-results/.*xml" -exec cp {} ~/micrometer/junit/ \;
+
+      - store_test_results:
+          path: ~/micrometer/junit/
+
+      - store_artifacts:
+          path: build/libs
```

Comment:
This isn't working as before, but I'm not actually sure what "artifacts" were being uploaded and what it effectively does. I don't see any "Artifacts" tab on our previous builds, but maybe that is a difference between CircleCI 1.0 and 2.0. Documentation: https://circleci.com/docs/2.0/artifacts/

I'm wondering if we should just delete this step. As far as I know, we do not have any use case to download artifacts from a specific build from CircleCI or view them in the CircleCI UI.

Reply:
If we wanted to store the JARs we would first need to collect them to a single folder like the JUnit reports.